velox/vector/FunctionVector.h

```diff
@@ -15,6 +15,7 @@
  */
 #pragma once
 
+#include <stdexcept>
```

Comment:
is this for using VELOX_UNSUPPORTED?

Reply:
thanks for catching this, VSCode keeps adding these unnecessary/incorrect includes, I missed this one.